DomainServices/Services/CustomerService.cs

```diff
@@ -0,0 +1,57 @@
+﻿using DomainModels;
+using DomainServices.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DomainServices.Services
+{
+    public class CustomerService : ICustomerService
+    {
+        private readonly List<Customer> _customers = new();
+
+        public virtual void Create(Customer customer)
+        {
+            var emailAlreadyExists = _customers.Any(_customer => _customer.Email == customer.Email);
+            if (emailAlreadyExists)
+                throw new ArgumentException("Email already exists");
+
+            var cpfAlreadyExists = _customers.Any(_customer => _customer.Cpf == customer.Cpf);
+            if (cpfAlreadyExists)
+                throw new ArgumentException("Cpf already exists");
+
+            customer.Id = _customers.LastOrDefault()?.Id + 1 ?? 1;
+
+            _customers.Add(customer);
+        }
+
+        public virtual void Delete(long id)
+        {
```

Comment:
[Comentário Acima](https://github.com/LuanFortunatoW/EverestBackendDotnet/pull/2/files#r1027900637)
```suggestion
        public void Delete(long id)
        {
```